module-2/backend/week_two.md

```diff
@@ -1,37 +1,93 @@
 ## Week Two - Module 2 Recap
 
-Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week (which is a TON - YOU are a web developer!!!). 
+Fork this repository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week (which is a TON - YOU are a web developer!!!).
 
-Note: When you're done, submit a PR. 
+Note: When you're done, submit a PR.
 
 1. At a high level, what is ActiveRecord? What does it do/allow you to do?
+ * ActiveRecord is a tool that works with relational databases that allows you to interact and query your database in specific ways.
+
 2. What kind of methods are `belongs_to`, and `has_many`? (i.e. class or instance) Give an example.
+ * they are class methods. For example a payload belongs_to a url but a url has_many payloads.
+
 3. What do they allow you to do?
+ * They allow you to add a class method that you can call on a single object. You can call Url.find(1).payloads (which will give you all the payload objects that belong to that specific Url)
+
```

Comment:
Yup! I might say 'single **instance** of an object', but I think you get the idea.